index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Medini Website Design</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h3 class="main_heading">My Logo</h3>
+        <img class="menubutton" src='./images/Union.svg' alt="menubutton">
```

Comment:
- [ ] Wrap the hamburg icon within `nav` tags

Reply:
Enclosed the hamburger icon within nav tag